controllers/teams/deleter.js

```diff
@@ -1,27 +1,41 @@
-const { Team } = require('../../databases/db');
+const { team} = require('../../databases/db');
 const { imgController, fileController } = require('../filesControllers'); 
 const path = require('path');
-
+const responses= require('../../utils/responses/responses');
+const Sticker= require('../../databases/db');
+const game= require('../../databases/db');
```

Comment:
```suggestion
const responses= require('../../utils/responses/responses');
const { Sticker, Game, Team }= require('../../databases/db');
```
Destructura bien la wea